recipes/essentials/all/test_package/conanfile.py

```diff
@@ -0,0 +1,36 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    test_type = "explicit"
+    requires = "spdlog/1.11.0", "fmt/9.1.0"
```

Comment:
to remove, these libs are not direct dependencies of test package

Reply:
Done. Requirements removed.